LogosLinuxInstaller.py

```diff
@@ -787,9 +795,9 @@ def make_skel(app_image_filename):
     verbose()
     print("skel64 done!")
 
-def check_commands(superuser_command, package_manager_command, os_name, missing_commands, packages, extra_info):
+def check_commands(commands):
```

Comment:
since the values for `superuser_command`, `package_manager_command`, and `packages` are used multiple times (at least twice, anyway), would it be okay to follow the practice of saving that data as environment variables (or even global variables?), rather than pass them around as arguments?

Reply:
If you think it would be better, I don't have a problem with it. But I do think it best if the script creates those variables rather than assumes a default, if that makes sense. :)

Reply:
These 2 commits address this:
- [3206f24](https://github.com/thw26/LogosLinuxInstaller/pull/7/commits/3206f249bcc1147a11fa40ccf03adfc18cd0c80d) (this a huge one where I implement the config.py module)
- [2d44ff1](https://github.com/thw26/LogosLinuxInstaller/pull/7/commits/2d44ff1972f592401528e7eb0341d192ce91bd91)

The `get_os()` and `get_package_manager()` functions are added to `set_default_config()`, which replaces `set_default_env()`. And their results are saved as part of the global config in config.py.